spec/integration/estimates_spec.rb

```diff
@@ -33,7 +27,7 @@
     )
 
     expect(flight_estimate.data.type).to eq 'flight'
-    expect(flight_estimate.data.mass_g).to eq 1_031_697
+    expect(flight_estimate.data.mass_g).to eq 1_000_622
```

Comment:
I confirmed this change is correct with Brendan

Reply:
As discussed, we will probably want to remove this type of tests from the SDK itself.